image_explorer/image_explorer.py

```diff
@@ -373,6 +373,8 @@ def _get_hotspots(self, xmltree, absolute_urls=False):
             if not hotspot.y.endswith('%'):
                 hotspot.y += 'px'  # px is deprecated as it is not responsive
 
+            hotspot.visited = hotspot.item_id in self.opened_hotspots:
```

Comment:
This is an extra : at the end of this line that's breaking this code.

Reply:
Sorry, apparently I only added that fix but did not commit. I've pushed the fix and made sure the tests were passing to be extra sure.
Thanks!

Reply:
@xirdneh No worries :-) 

Could you add a version bump as well? After the I will merge it.